src/main/kotlin/net/lumalyte/lg/infrastructure/listeners/GuildDisbandedListener.kt

```
@@ -1,24 +1,43 @@
             player.sendMessage("§c✗ Your guild §f${event.guild.name} §chas been disbanded.")
         }
+
+        // 2. Remove all Party channels associated with this guild
+        try {
+            val parties = partyService.getActivePartiesForGuild(event.guild.id)
+            parties.forEach { party ->
+                if (partyService.deleteParty(party.id)) {
+                    logger.info("Removed guild channel ${party.name} (${party.id}) for disbanded guild ${event.guild.name}")
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)